dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/test-applications/loadonstartup/src/com/ibm/ws/jaxrs20/cdi12/fat/loadonstartup/StartupFilter1.java

```diff
@@ -13,12 +13,13 @@
 import java.io.IOException;
 
 import javax.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.RequestScoped;
 import javax.inject.Inject;
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerRequestFilter;
 
-@ApplicationScoped
+// Change from ApplicationScoped to RequestScoped to verify the fix for issue 9976
+@RequestScoped
```

Comment:
Instead of changing this test, could we duplicate the test - and use one with App scope and the other with Request scope?